folding-schemes/src/folding/nova/mod.rs

```diff
@@ -272,7 +273,7 @@ where
             poseidon_config: pp.poseidon_config.clone(),
             cs_params: pp.cs_params.clone(),
             cf_cs_params: pp.cf_cs_params.clone(),
-            F,
+            F: F_clone,
```

Comment:
Maybe the `F_clone` is not needed and line 246 can just be removed and line 276 can be just
```suggestion
            F,
```
How would you see it?